examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Deploy tfx model
+name: Deploy Vertex AI tfx model
```

Comment:
@javiergp should we change the name to deploy-tfx.yml.TEMPLATE?

Reply:
I'll add this in the TODO list